### PR TITLE
OSD-13649: updaed MUO configmap to ignore etcdGRPCRequestsSlow alert

### PR DESCRIPTION
**Pull request by shitaljante**
### What this PR does / why we need it?
- etcdGRPCRequestsSlow alert auto-resolve by self in some time within 5-10 mins.
- This alert has been converted into warning as it is self resolvable in nature. 
- MUO can block the upgrade considering this alert as a critical raised by cluster. 
- To avoid the upgrade cancellations due to this scenario, updated the MUO configmap to ignore etcdGRPCRequestsSlow in pre/post upgrade checks. 

### Which Jira/Github issue(s) this PR fixes?
https://issues.redhat.com/browse/OSD-13649

### CODE REVIEW

**Comment by mrbarge** [comment]
We'll also want
`managed-upgrade-operator-config/10-managed-upgrade-operator-configmap.yaml`
updated as well, as that's the version that goes out to all clusters 4.8 and above.

File: deploy/managed-upgrade-operator-config/4.7/10-managed-upgrade-operator-configmap.yaml [hunk 1]
```diff
@@ -52,6 +52,7 @@ data:
       - ThanosRuleSenderIsFailingAlerts
       - ThanosRuleHighRuleEvaluationFailures
       - ThanosNoRuleEvaluations
+      - etcdGRPCRequestsSlow
```
Comment (mrbarge) [inline deploy/managed-upgrade-operator-config/4.7/10-managed-upgrade-operator-configmap.yaml:55]: When adding an alert to this list, please add a leading comment line (`# ....`) that links to the Jira ticket where this decision was made.

**Comment by shitaljante** [comment]
> We'll also want `managed-upgrade-operator-config/10-managed-upgrade-operator-configmap.yaml` [0] updated as well, as that's the version that goes out to all clusters 4.8 and above.
> 
> [0] https://github.com/openshift/managed-cluster-config/blob/master/deploy/managed-upgrade-operator-config/10-managed-upgrade-operator-configmap.yaml

Sure, I will add. Do we need this change for lower versions i.e 4.5 and 4.6 as well ?

File: deploy/managed-upgrade-operator-config/4.7/10-managed-upgrade-operator-configmap.yaml (hunk 1 above)
Comment (shitaljante) [reply deploy/managed-upgrade-operator-config/4.7/10-managed-upgrade-operator-configmap.yaml]: Added change, Thanks :) @mrbarge

**Pull request merged**